Styles2.css

```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: 'Times New Roman', Times, serif, sans-serif;
+    font-size: x-large;
+    background-color: bisque;
+}
+
+.container {
+    max-width: 960px;
+    margin: 0 auto;
+    padding: 20px;
+}
+
+/* Global styles */
+body, html {
```

Comment:
it looks like there are several styles that are repeated throughout this file. consider doing a lookover and see where any can be removed.